Subject: Don't require libdb

--- pam-1.5.2.orig/debian/control	2022-08-24 09:36:02.912263961 +0000
+++ pam-1.5.2/debian/control	2022-08-24 09:36:33.505703954 +0000
@@ -4,9 +4,9 @@ Priority: optional
 Uploaders: Sam Hartman <hartmans@debian.org>
 Maintainer: Steve Langasek <vorlon@debian.org>
 Standards-Version: 4.6.0
-Build-Depends: debhelper-compat (= 13), quilt, flex, libdb-dev, libselinux1-dev [linux-any], po-debconf, dh-autoreconf, autopoint, libaudit-dev [linux-any] <!stage1>, pkg-config, libfl-dev, libfl-dev:native, docbook-xsl, docbook-xml, xsltproc, libxml2-utils, w3m
+Build-Depends: debhelper-compat (= 13), quilt, flex, libselinux1-dev [linux-any], po-debconf, dh-autoreconf, autopoint, libaudit-dev [linux-any] <!stage1>, pkg-config, libfl-dev, libfl-dev:native, docbook-xsl, docbook-xml, xsltproc, libxml2-utils, w3m
 Build-Conflicts-Indep: fop
-Build-Conflicts: libdb4.2-dev, libxcrypt-dev
+Build-Conflicts: libxcrypt-dev
 Vcs-Browser: https://salsa.debian.org/vorlon/pam
 Vcs-Git: https://salsa.debian.org/vorlon/pam.git
 Homepage: http://www.linux-pam.org/
--- pam-1.5.2.orig/debian/rules	2022-08-24 09:37:06.195263496 +0000
+++ pam-1.5.2/debian/rules	2022-08-24 09:41:55.703590089 +0000
@@ -30,6 +30,7 @@ override_dh_auto_configure:
 	dh_auto_configure -- --enable-static --enable-shared \
 		--libdir=/lib/$(DEB_HOST_MULTIARCH) \
 		--enable-isadir=/lib/security \
+		--enable-db=no \
 		--with-systemdunitdir=/usr/lib/systemd/system \
 		--disable-nis \
 		$(CONFIGURE_OPTS)
